didx-xyz/yoma-mobile
# feat(Skills): save user skills

## Scope [YOMA-456](https://yomaworld.atlassian.net/browse/YOMA-456)

# GIF TAX
![](https://media.giphy.com/media/6R2mLi910HL4VXFwOG/giphy.gif)

## Work Done
- wiring up component to properly submit the form
- dispatching values, and saving through middleware
- handling return by updating the current skills with the new skills added.

## Steps to Test
- go to My Skills widget, select Add.
- Select a few skills (or 1) and save
- see that you get a successful toast and the skills are displayed in your listings.
